sdl2promela/sdl2promela.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import typing
+from typing import List
+import logging
+import opengeode
+import sys
+import traceback
+
+from opengeode import ogAST
+from .sdl import model as sdlmodel
+from .promela import model as promelamodel
+from .promela import generator as promelagenerator
+from . import translator
+from . import __version__
+
+log = logging.getLogger("sdl2promela")
+
+def read_process(sdl_files : List[str]) -> ogAST.Process:
+    try:
+        ast, warnings, errors = opengeode.parse(sdl_files)
+    except IOError as error:
+        log.error("IOError:" + error)
+        traceback.print_exc()
+        sys.exit(1)
+    except:
```

Comment:
it is generally not recommended to "catch everything" - same issue as with `catch (...)` in C++ (think about "out of memory" exception and trying to log something)

Reply:
Except this is actually my whole intent. I have no idea what may go wrong, so I catch everything and prnit error information

Reply:
but this way you risk "python.exe terminated unexpectedly" instead of a nice stack trace.
Remember, it caches also `sys.exit()`  or `Ctrl+C` etc. Even `except Exception` is better (although still considered "not the best prectice").

Reply:
" doesn't catch BaseException or the system-exiting exceptions SystemExit, KeyboardInterrupt and GeneratorExit".
Fair point. Let it be except Exception.